### PR TITLE
Release google-cloud-vision 0.31.1

### PR DESCRIPTION
**Pull request by quartzmo**
* Remove focus from acceptance tests (#2490)

This pull request was generated using releasetool.

### CODE REVIEW

**Comment by quartzmo** [comment]
I created this PR to test releasetool development. WIP. Do not merge.

**Comment by blowmage** [comment]
Vision doesn’t need to be released. The files that changes are not included in the gem.

File: google-cloud-dlp/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release History
 
+### 0.6.3 / 2018-09-28
+
+* Update google-cloud-dlp generated files (#2481)
+* Re-generate library using google-cloud-dlp/synth.py (#2478)
```
Comment (blowmage) [inline google-cloud-dlp/CHANGELOG.md]: We need to do a better job at updating the merge commit to make a meaningful description of the changes. IMO.

**Comment by quartzmo** [comment]
Purpose was releasetool development, closing.

**Pull request closed without merging**